coding_convention.md

```diff
@@ -0,0 +1,11 @@
+This file describes this project's coding conventions. There are many items to add here, but we will start
+small and will improve over time.
```

Comment:
I think it's enough to just have `This file describes this project's coding conventions.`

If there are more things to add, they should be there, and if they are not, a dev can't assume they 'should exist'

Sorry for being picky x)

Reply:
You are right, fixed!